### DISCUSSION TITLE
Dev Server Port Changing Randomly on Autoreload

### DISCUSSION THREAD

**Discussion by kpodp0ra** [teableio/teable#1030] [General]
I've been experiencing an issue during development, and I'm wondering if anyone else has encountered the same. Sometimes, when I'm working with autoreload on my dev server, the port will randomly switch from :3000 to :3002. After restarting, it usually reverts back to :3000, but it's pretty inconsistent.

Has anyone else faced this issue? Is there a known workaround or configuration tweak that can lock the port to avoid this random switching?

**Comment by tea-artist** [comment]
Did port change after you modified the backend code and auto reloaded?

**Reply by kpodp0ra** [reply]
Yes
